### PR TITLE
add delete pipeline test case

### PR DESCRIPTION
**Pull request by HarrisChu**

### CODE REVIEW

**Review by yixinglu — COMMENTED** [review]
please format feature files:

```shell
$ cd tests
$ make fmt
```

**Comment by HarrisChu** [comment]
> please format feature files:
> 
> ```shell
> $ cd tests
> $ make fmt
> ```

run reformat-gherkin manually

**Comment by Aiee** [comment]
> > please format feature files:
> > ```shell
> > $ cd tests
> > $ make fmt
> > ```
> 
> run reformat-gherkin manually

Please re-check the format and push the new commit again.
Also, don't forget to sign the [Contributor License Agreement ](https://cla-assistant.io/vesoft-inc/nebula-graph?pullRequest=888)

File: tests/tck/features/delete/DeleteVertex.IntVid.feature [hunk 1]
```diff
@@ -196,4 +196,23 @@ Feature: Delete int vid of vertex
       """
     Then the result should be, in any order:
       | VertexID | player.name | player.age |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == 'Kyle Anderson'
+      """
+    Then the result should be, in any order:
+      | VertexID             |
+      | -8310021930715358072 |
```
Comment (yixinglu) [inline tests/tck/features/delete/DeleteVertex.IntVid.feature:205]: `hash("Kyle Anderson")`

**Pull request closed without merging**